RA_Integration/RA_Core.cpp

```diff
@@ -234,32 +235,32 @@ API int CCONV _RA_Shutdown()
 
 	RAWeb::RA_KillHTTPThreads();
 
-	if (g_AchievementsDialog.GetHWND() != nullptr)
+	if ( g_AchievementsDialog.GetHWND() != nullptr )
```

Comment:
Maybe it's time to define a coding standard. I've seen all of the following:
```
if( a )
if (a)
if ( a )
```

Reply:
Personally, I prefer the middle one, probably because it's what I've used professionally.

Reply:
The latest VS has some fairly solid default formatting rules, I've been trying to stick to them lately without reformatting unnecessarily